lib/features/dashboard/account/account_view.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_modular/flutter_modular.dart'
+    hide ModularWatchExtension;
+import 'package:live_score/core/constants/routes.dart';
+import 'package:live_score/core/extensions/localization/app_localizations_context.dart';
+import 'package:live_score/core/theme/app_theme.dart';
+import 'package:live_score/core/ui/dashboard/account/log_out_button.dart';
+import 'package:live_score/features/dashboard/cubit/dashboard_cubit.dart';
+
+class AccountView extends StatelessWidget {
+  const AccountView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final cubit = context.read<DashboardCubit>();
+    cubit.loadUser();
+    return BlocConsumer<DashboardCubit, DashboardState>(
+        listener: (context, state) {
+      if (state is LoggedOut) {
```

Comment:
Please move the whole listener to a function called `void _accountViewListener` or something along these lines